online-banking-cz/src/main/java/com/adyen/checkout/onlinebankingcz/OnlineBankingCZView.kt

```diff
@@ -115,31 +112,8 @@ class OnlineBankingCZView @JvmOverloads constructor(
         binding.textInputLayoutOnlineBanking.isEnabled = enabled
     }
 
-    private fun styleTermsAndConditions(string: String) {
-        val firstIndex = string.indexOf("%#", 0, ignoreCase = true)
-        val secondIndex = string.indexOf("%#", firstIndex + 2, ignoreCase = true)
-
-        val newString = string.replace("%#", "", ignoreCase = true)
-
-        val spannable = SpannableString(newString).apply {
-            setSpan(
-                ForegroundColorSpan(ResourcesCompat.getColor(resources, R.color.textColorLink, null)),
-                firstIndex,
-                secondIndex - 2,
-                Spannable.SPAN_EXCLUSIVE_EXCLUSIVE
-            )
-            setSpan(
-                UnderlineSpan(),
-                firstIndex,
-                secondIndex - 2,
-                Spannable.SPAN_EXCLUSIVE_EXCLUSIVE
-            )
-        }
-
-        binding.textviewTermsAndConditions.text = spannable
-    }
-
     companion object {
+        private const val REPLACEMENT_TOKEN = "%#"
```

Comment:
This should probably be in the same file as `formatStringWithHyperlink` since we will use the same token everywhere.